.github/workflows/mobile-deploy.yml

```
@@ -310,8 +315,40 @@
+          NEW_VERSION=$(jq -r .version version.json)
+          echo "Updating package.json to version $NEW_VERSION"
+          # Use yarn to update package.json and the lockfile
+          yarn version --new-version $NEW_VERSION --no-git-tag-version
+
+      - name: Commit version files
+        if: success() && github.event.inputs.test_mode != 'true'
+        run: |
+          cd ${{ github.workspace }}
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)